padd.sh

```diff
@@ -1370,6 +1370,58 @@ NormalPADD() {
   done
 }
 
+Update() {
+    GetPADDInformation
+
+   if [ "${padd_out_of_date_flag}" = "true" ]; then
+        echo "${full_status_update}"
+
+        padd_script_path="$0"
+        padd_script_file_name="padd.sh"
+        padd_script_directory="$(pwd -P)"
+
+        # Resolve potential symlinks until the script path is found
+        while :; do
+            if [ ! -L "${padd_script_path}" ] && [ ! -e "${padd_script_path}" ]; then
+                echo "${check_box_bad} Cannot locate script to update"
+                exit 1
+            fi
+            if ! cd "$(dirname -- "${padd_script_path}")"; then
+                echo "${check_box_bad} Cannot locate script to update"
+                exit 1
+            fi
+            pad_script_file_name="$(basename -- "${padd_script_path}")"
+            [ "${padd_script_file_name}" = '/' ] && "${padd_script_file_name}"=''
```

Comment:
I think you meant this:

```suggestion
            [ "${padd_script_file_name}" = '/' ] && padd_script_file_name=''
```

Reply:
Your are right, typo / copy-paste failed me.